### PR TITLE
update @polkadot/apps-config

### PR DESCRIPTION
**Pull request by xlc**
Solves compatibility issue with Karura and likely for other networks as well.

### CODE REVIEW

**Review by emostov — APPROVED** [review]

**Review by TarikGul — APPROVED** [review]
lgtm. Thanks!

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
   },
   "dependencies": {
     "@polkadot/api": "^5.8.4-1",
-    "@polkadot/apps-config": "^0.95.1",
+    "@polkadot/apps-config": "^0.95.2-104",
```
Comment (jsdw) [inline package.json:53]: Should this be listed in the resolutions as well with the other polkadot deps?

Comment (dvdplm) [reply package.json:53]: Looks like `0.95.2-106` is out too. Worth bumping here?

**Comment by xlc** [comment]
Can we have this merged and released to fix the compatibility issue with current release?

**Comment by jsdw** [comment]
Given that we aim to do regular releases on Monday which will lead to bumping this (and other) dependencies as needed to keep up, I'd suggest at this point that we perform the update+release on Monday, which will hopefully address the compatibility issues that you're facing :)

**Comment by jsdw** [comment]
I'll be releasing this today, so if all goes well this should be merged and released shortly, and I'll make sure that the version bumps in this PR are a part of that and close this for now :)

**Comment by jsdw** [comment]
FYI, v9.1.9 has now been released, which bumps `@polkadot/apps-config` to `^0.95.2-114`, so hopefully that will resolve your issue @xlc :)

**Pull request closed without merging**